source/Clean.js

```diff
@@ -132,26 +137,28 @@ var filterSpanAttributes = function(span){
     return filterAttributes(span, whiteList)
 }
 
-//NATE: I lke the stylesRewriters, we should have sane defaults for all the elements as a first pass, and then
+var replaceStyles = function ( node, parent ) {
+  //NATE: TODO: whitelist of classes for span
+  span.removeAttribute("style")
+  filterSpanClasses(span)
+  filterSpanAttributes(span)
+  return span
+  // NATE: I want to leave one line of the old code in as a reminder, this is the line that was causing
+  // one span to get broken out into many spans, but it was kind of clever and we might want to use the idea
+  // at a later time.  It looked at if something had a large font or a certain color and tried to guess what the
+  // element was and add a semantic class to the span.  For each attribute there was a separate class.  Although
+  // we don't want a bunch of spans with classes, we might want to re-order a span with certain attributes into
+  // more sensible elements
+  // for ( attr in spanToSemantic )
+};
+//NATE: I like the stylesRewriters, we should have sane defaults for all the elements as a first pass, and then
 // any additional complicated filtering can be done by registering filters with squire that will be executed during
 // insertHTML
 var stylesRewriters = {
-    SPAN: function ( span, parent ) {
-        //NATE: TODO: whitelist of classes for span
-        span.removeAttribute("style")
-        filterSpanClasses(span)
-        filterSpanAttributes(span)
-        return span
-        // NATE: I want to leave one line of the old code in as a reminder, this is the line that was causing
-        // one span to get broken out into many spans, but it was kind of clever and we might want to use the idea
-        // at a later time.  It looked at if something had a large font or a certain color and tried to guess what the
-        // element was and add a semantic class to the span.  For each attribute there was a separate class.  Although
-        // we don't want a bunch of spans with classes, we might want to re-order a span with certain attributes into
-        // more sensible elements
-        // for ( attr in spanToSemantic )
-    },
+    SPAN: replaceStyles,
     STRONG: replaceWithTag( 'B' ),
     EM: replaceWithTag( 'I' ),
+    INS: replaceWithTag( 'U' ),
     STRIKE: replaceWithTag( 'S' ),
     P: replaceWithTag('DIV'),
```

Comment:
by the way, if we want to swap p tags for divs, we invert this line as the major change